components/PralineList/PralineList.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import useSWR from "swr";
 import Image from "next/image";
 
-export default function PralineList({ onSelectPraline }) {
+export default function PralineList({ onSelectPraline, buttonName }) {
```

Comment:
You could solve this with `children` too. Anything that is passed between the opening and closing brackets of `PralineList` gets passed down as `children`.
```suggestion
export default function PralineList({ onSelectPraline, children}) {
```
Don't forget to change the way you use the component too.